environment-setup/ansible/TP2/roles/kubernetes/files/generate-user-k8s-config.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -eu
+is_file_exists(){
```

Comment:
Any call to this function can be replaced with `[[ -f $file ]]` directly (the function does not do anything else).